### ISSUE TITLE
Where does Streisand place it logs?

### ISSUE THREAD

**Issue by vonHabsi** [jlund/streisand-discussions#189]
I am looking for some log file to see where Streisand (openvpn) places its logs and I can't find any.

Does that mean I have to add logging to the relevant server.conf?

**Comment by alexqem** [comment]
Yeah, what's the point of the app if it's having a problem with the connection and we can't see the app's logs?
